internal/commands/gen/tfenv.go

```diff
@@ -152,7 +152,7 @@ func GenerateTerraformFiles(region, profile, env, namespace, stateBucketName str
 		AWS_REGION:        region,
 		EC2_KEY_PAIR_NAME: fmt.Sprintf("%v-%v", env, namespace),
 		ROOT_DOMAIN_NAME:  viper.GetString("infra.terraform.root_domain_name"),
-		TAG:               env,
+		TAG:               fmt.Sprintf("%s-latest", env),
```

Comment:
Can you please comment why are we changing this and not setting `TAG` in the test itself?
I think the reason is b/c we don't have ability  to pass `TAG` at all... Maybe that should be the change instead?